packages/footer/package.json

```diff
@@ -19,7 +19,7 @@
       "@chakra-ui/react": ">=2",
       "@core-ds/primitives": ">2.4",
       "@emotion/react": ">=11.0.0 <12.0.0",
-      "framer-motion": ">=3.0.0 <4.0.0 || >=4.0.0 <5.0.0 || >=5.0.0 <6.0.0 || >=6.0.0 <7.0.0",
+      "framer-motion": ">=4.0.0 <11.0.0",
```

Comment:
I changed the minimum version to 4 cause I think that's the minimum version supported by Chakra